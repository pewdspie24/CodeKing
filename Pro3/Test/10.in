100
86 83135878
384913 707329
591722 820117
767741 951633
42054 421609
931390 475614
867276 6145
474244 968300
469237 703656
167813 233794
223331 270171
503902 267091
271779 739793
932489 875039
339510 316823
944706 74732
653165 407109
803133 303063
809135 396045
616737 902910
593194 655747
491937 309970
37575 337469
997159 809792
418232 103639
120961 889562
967453 257984
758249 130244
272085 171207
350550 106511
51267 572185
236703 185760
296469 676657
875474 90924
604264 597394
728713 708997
891654 959285
839462 632667
706363 681814
342239 785601
120816 730305
916521 793644
488118 73976
4910 258656
83058 64087
563161 48009
333748 549447
845318 900050
552386 258624
900726 701012
450204 786504
454859 79247
645247 872031
594203 890040
885940 160762
90272 14107
980670 540392
259148 708344
482775 392695
863355 897586
461530 663448
13687 541037
132067 388660
621920 387431
590279 433902
657371 275237
18378 357692
49237 654932
240837 543101
38392 550936
111936 495162
376054 208273
405279 58213
503663 271747
149279 675983
276019 156115
798996 474669
257097 953891
842422 40917
233888 490438
850750 55825
511501 568489
202339 316180
145508 385786
276259 499299
88710 99295
673001 437402
