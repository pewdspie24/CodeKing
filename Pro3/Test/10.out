96
95
94
92
91
90
89
88
87
86
84
83
82
81
80
79
78
76
75
74
73
72
71
70
68
67
66
65
64
63
62
60
59
58
57
56
55
54
52
51
50
49
48
47
45
44
43
42
41
40
39
37
36
35
34
33
32
31
29
28
27
26
25
24
23
21
20
19
18
17
16
15
13
12
11
10
9
8
7
5
4
3
2
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
